cdc/scheduler/util/table_set.go

```diff
@@ -35,6 +35,12 @@ type TableRecord struct {
 // The meaning of its value can be defined by the user of TableSet.
 type TableStatus int32
 
+const (
+	AddingTable = TableStatus(iota)
```

Comment:
Could you start enums at 1?

See https://github.com/uber-go/guide/blob/master/style.md#start-enums-at-one

Reply:
Does the comment at line 35 need update?